app/assets/javascripts/govuk_publishing_components/lib/govspeak/barchart-enhancement.js

```diff
@@ -10,16 +10,26 @@ window.GOVUK = window.GOVUK || {};
   }
 
   BarchartEnhancement.prototype.init = function () {
-    // the not selectors are protection against initialising twice since that
+    // this is protection against initialising twice since that
     // causes quite a mess. The not .mc-chart is because it creates a second
     // .js-barchart-table element with .mc-chart and then the
     // .js-barchart-table-init is given when we initialise
-    var $barcharts = this.$element.find('.js-barchart-table:not(.mc-chart):not(.js-barchart-table-init)')
-    $barcharts.each(function () {
-      var $table = $(this)
-      $.magnaCharta($table, { toggleText: 'Change between chart and table' })
-      $table.addClass('js-barchart-table-init')
-    })
+    // we'd use :not selectors but for IE8, previously worked before jquery removal
+    var $barchartsOriginal = this.$element.querySelectorAll('.js-barchart-table')
+    var $barcharts = []
+
+    for (var j = 0; j < $barchartsOriginal.length; j++) {
+      var styles = $barchartsOriginal[j].className
+      if (styles.indexOf('mc-chart') === -1 && styles.indexOf('js-barchart-table-init') === -1) {
+        $barcharts.push($barchartsOriginal[j])
+      }
+    }
+
+    for (var i = 0; i < $barcharts.length; i++) {
+      var $table = $barcharts[i]
+      new GOVUK.Modules.MagnaCharta().start($table, { toggleText: 'Change between chart and table' })
+      $table.classList.add('js-barchart-table-init')
```

Comment:
Similar to previous comment about better Internet Explorer support:

```suggestion
      $table.className = $table.className + ' js-barchart-table-init'
```